### PR TITLE
Reorganize the top level documentation

### PR DESCRIPTION
**Pull request by blink1073**
cc @willingc  @jasongrout @Carreau 

Separates out the contributing and release instructions from the main readme, and adds some of the content from the notebook contributing instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,146 +58,7 @@ the browser.
...
-## Bundle for the Browser
-
-Follow the package install instructions first.
-
-Any bundler that understands how to `require()` files with `.js` and `.css`
-extensions can be used with this package.
-
-**Note:** This npm module is fully compatible with Node/Babel/ES6/ES5. Simply
-omit the type declarations when using a language other than TypeScript.
+See the [Contributing Guidlines](http://github.com/jupyterlab/CONTRIBUTING.rst).
```
Comment (willingc) [inline README.md]: CONTRIBUTING.rst -> CONTRIBUTING.md

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+
+Making a JupyterLab release
+===========================
+
+This document guides a contributor through creating a release of JupyterLab.
+
+Check installed tools
+---------------------
+
+Review ``CONTRIBUTING.rst``. Make sure all the tools needed to generate the
```
Comment (willingc) [inline RELEASE.md]: CONTRIBUTING.rst -> CONTRIBUTING.md

**Comment by willingc** [comment]
Yay docs! Overall, looks good 👍

**Comment by blink1073** [comment]
Fixed!

**Comment by afshin** [comment]
👍

File: README.md [hunk 3]
```diff
@@ -52,146 +58,7 @@ the browser.
...
-## Bundle for the Browser
-
-Follow the package install instructions first.
-
-Any bundler that understands how to `require()` files with `.js` and `.css`
-extensions can be used with this package.
-
-**Note:** This npm module is fully compatible with Node/Babel/ES6/ES5. Simply
-omit the type declarations when using a language other than TypeScript.
+See the [Contributing Guidelines](http://github.com/jupyterlab/CONTRIBUTING.md).
```
Comment (willingc) [inline README.md]: Fix wasn't as simple as `.md`. Link needs updating.

Also, we added navigation to the top of the JupyterHub README the other day. You may like it for some of these `.md` files for navigation: https://github.com/jupyterhub/jupyterhub/blob/master/README.md

Comment (afshin) [inline README.md]: 👍 on the top-level navigation, it'll help with cross-pollination of contributors.

**Comment by blink1073** [comment]
I'll finish updating this tomorrow.

**Comment by blink1073** [comment]
@willingc, updated to more closely match JupyterHub, thanks for the prompt and stellar example!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,197 @@
...
+If you use ``pip``` you can install it as:
+
+```bash
+pip install notebook
+```
+
+Once you have installed the dependencies mentioned above, use the following
+steps::
+
+    git clone https://github.com/<your-github-username>/jupyterlab.git
```
Comment (Carreau) [inline CONTRIBUTING.md]: You need to fork it first if you do that :-) Should we just tell people to clone form jupyter/jupyterlab ?

**Comment by Carreau** [comment]
+1, even to merge as is and refine !

Thanks !

**Comment by willingc** [comment]
Thanks @blink1073 and team. Taking @carreau's suggestion.

**Pull request merged**